docs/pages/components/select.md

```diff
@@ -414,19 +414,17 @@ The disabled state can also be achieved by adding the `.is-disabled` class or th
 To make the `Select` component read-only, the readonly attribute needs to be added to the `fd-select__control` element. 
 This can also be done by adding the `.is-readonly` class or the `aria-readonly="true"` attribute.
 
-{% capture disabled-select %}
+{% capture readonly-select %}
 <div class="fd-popover">
     <div class="fd-popover__control" aria-expanded="false" aria-haspopup="false" aria-readonly="true" readonly>
         <div class="fd-select">
-            <div class="fd-select__control" role="button" aria-expanded="false" aria-haspopup="false" aria-readonly="true" readonly>
-                Select
-           </div>
+            <div class="fd-select__control" role="button" aria-expanded="false" aria-label="Select" aria-haspopup="false" aria-readonly="true" readonly></div>
```

Comment:
Should this have an aria-label like I've added here because the placeholder is removed in readOnly state? @meganaconley @jacobdevera @prsdthkr

Reply:
Can you show where it says not to include a placeholder? I'm looking at https://experience.sap.com/fiori-design-web/select/#readonly and it seems fine as-is.

I think the confusion may be that it seems to be a placeholder when it should actually be a pre-selected value.

Reply:
For everyone else - sent wiki link privately as it's internal :)

Reply:
We can set this one to be a selected item for the docs though?

Reply:
Yeah, the text here could be changed to make clear that it's a selected item rather than a placeholder.